content/_help/verify-your-identity/accepted-state-issued-identification._es.md

```diff
@@ -7,7 +7,7 @@ order: 2
 
 Por el momento, solo se aceptan los siguientes documentos de identidad estatales:
 
-Licencia de conducir válida en los 50 estados, el Distrito de Columbia (DC) y otros territorios de Estados Unidos (Guam, Islas Vírgenes de Estados Unidos, Islas Marianas y Puerto Rico)
+Licencia de conducir válida en los 50 estados, el Distrito de Columbia (DC) y otros territorios de Estados Unidos (Guam, Islas Vírgenes de Estados Unidos, Samoa Estadounidense, Islas Marianas y Puerto Rico)
```

Comment:
In #1041, this was called "Samoa Americana". Which one is correct?

Reply:
Nice catch! Got confirmation that translation is what we are using across the site. Updated.